services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -665,6 +674,20 @@ input ContentTeaserInput {
 
 input ContentBodyInput {
   mutation: ContentMutation = Website
+  imageAttrs: EmbeddedImageAttrsInput = {
+    w: 1280,
+    fit: "max",
+    auto: "format,compress"
+  }
+}
+
+input EmbeddedImageAttrsInput {
+  "The (max) width of the embedded image"
```

Comment:
This isn't actually the max width. It will force the image to specify width regardless of the images actual size